x-pack/legacy/plugins/ml/public/settings/calendars/edit/directive.js

```diff
@@ -66,9 +61,7 @@ module.directive('mlNewCalendar', function ($route) {
 
       ReactDOM.render(
         <I18nContext>
-          <NavigationMenuContext.Provider value={{ chrome, timefilter, timeHistory }}>
-            <NewCalendar {...props} />
-          </NavigationMenuContext.Provider>
+          <NewCalendar {...props} />
```

Comment:
i'm a bit confused by this change. was `NavigationMenuContext.Provider` not really necessary  here in the last change because `chrome`, `timefilter`, `timeHistory` aren't being used by the calendars edit component?

Reply:
`NavigationMenuContext` is now `UiContext` and we're adding `chrome` etc. as the default value in the file where the context is instantiated. This means we can skip `<*.Provider>` in the JSX part and React will automatically fall back to the default value whereever we use `useContext...`.